blocks/core/ff_container/ff_container-dashboard-section/ff_container-dashboard-section.less

```diff
@@ -0,0 +1,32 @@
+//header
+.ff_container-dashboard-section__header {
+    padding: @ff_size_control_vertical;
+    background-color: @ff_colour_control_bg;
+    border: 1px solid @ff_colour_control_border;
+    border-top-left-radius: @ff_size_control_rounding;
+    border-top-right-radius: @ff_size_control_rounding;
+}
+
+.ff_container-dashboard-section__title {
+    font-size: 1rem;
```

Comment:
Do we need rem-fallbacks, or were we going to try something else for font sizes?